### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.hover\:bg-orange-700)

### ISSUE THREAD

**Issue by eknowles** [SomoGlobal/whizz-kidz-website#1]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://dequeuniversity.com/rules/axe/4.0/color-contrast?application=msftAI))
#### Target application
[Trusts, Statutory & Lottery](http://localhost:3000/charity/trusts-statutory-and-lottery)
#### Element path
.hover\:bg-orange-700
#### Snippet
`<a class="px-3 py-2 hover:text-white uppercase font-bold text-sm hover:bg-orange-700 text-white bg-orange-600" href="/charity">The Charity</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.38 (foreground color: #ffffff, background color: #dd6b20, font size: 10.5pt (14px), font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 85.0.4183.83
This accessibility issue was found using Accessibility Insights for Web 2.21.1 (axe-core 4.0.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
